deployment/helm/values.yaml

```diff
@@ -132,6 +203,27 @@ debugEcho:
     targetPort: 6052
     # protocol is the service protocol of the instance service
     protocol: TCP
+  # discovery defines a set of values for a debugEcho discovery handler DaemonSet
+  discovery: 
+    # enabled defines whether discovery handler pods will be deployed in a modular Agent scenario
+    enabled: false
```

Comment:
i wonder if discovery.enabled can default to agent.modular?

Reply:
Then all of the discovery handler pods would be deployed. I was thinking that would be a rare scenario, and that if folks wanted them all, they could use the non-slim agent. This discovery.enabled is the switch for deploying the discovery handler Daemonset for that particular protocol. I was wondering if `agent.modular.enabled` (now `agent.slim.enabled`) is necessary. Right now it acts kind of as an extra check, making people think twice before they deploy a discovery handler Daemonset, but when we have our slim build set up, it will be what triggers using that image.

Reply:
good point.